packages/svelte2tsx/svelte-jsx.d.ts

```diff
@@ -1,5 +1,286 @@
 /// <reference lib="dom" />
-/* eslint @typescript-eslint/no-unused-vars: off */
+
+declare namespace svelte {
+  type NativeElement = HTMLElement;
+
+  // TypeScript SVGElement has no `dataset` (Chrome 55+, Firefox 51+).
+  type Element = NativeElement & {
+    dataset: DOMStringMap;
+  };
+
+  //
+  // Event Handler Types
+  // ----------------------------------------------------------------------
+  type EventHandler<E extends Event = Event, T extends EventTarget = HTMLElement> =
+    svelte.JSX.EventHandler<E,T>;
+
+  type ClipboardEventHandler<T extends EventTarget> = svelte.JSX.ClipboardEventHandler<T>;
+  type CompositionEventHandler<T extends EventTarget> = svelte.JSX.CompositionEventHandler<T>;
+  type DragEventHandler<T extends EventTarget> = svelte.JSX.DragEventHandler<T>;
+  type FocusEventHandler<T extends EventTarget> = svelte.JSX.FocusEventHandler<T>;
+  type FormEventHandler<T extends EventTarget> = svelte.JSX.FormEventHandler<T>;
+  type ChangeEventHandler<T extends EventTarget> = svelte.JSX.ChangeEventHandler<T>;
+  type KeyboardEventHandler<T extends EventTarget> = svelte.JSX.KeyboardEventHandler<T>;
+  type MouseEventHandler<T extends EventTarget> = svelte.JSX.MouseEventHandler<T>;
+  type TouchEventHandler<T extends EventTarget> = svelte.JSX.TouchEventHandler<T>;
+  type PointerEventHandler<T extends EventTarget> = svelte.JSX.PointerEventHandler<T>;
+  type UIEventHandler<T extends EventTarget> = svelte.JSX.UIEventHandler<T>;
+  type WheelEventHandler<T extends EventTarget> = svelte.JSX.WheelEventHandler<T>;
+  type AnimationEventHandler<T extends EventTarget> = svelte.JSX.AnimationEventHandler<T>;
+  type TransitionEventHandler<T extends EventTarget> = svelte.JSX.TransitionEventHandler<T>;
+  type MessageEventHandler<T extends EventTarget> = svelte.JSX.MessageEventHandler<T>;
+
+  // See CSS 3 CSS-wide keywords https://www.w3.org/TR/css3-values/#common-keywords
+  // See CSS 3 Explicit Defaulting https://www.w3.org/TR/css-cascade-3/#defaulting-keywords
+  // "all CSS properties can accept these values"
+  type CSSWideKeyword = svelte.JSX.CSSWideKeyword;
+
+  // See CSS 3 <percentage> type https://drafts.csswg.org/css-values-3/#percentages
+  type CSSPercentage = svelte.JSX.CSSPercentage;
+
+  // See CSS 3 <length> type https://drafts.csswg.org/css-values-3/#lengths
+  type CSSLength = svelte.JSX.CSSLength;
+
+  // This interface is not complete. Only properties accepting
+  // unit-less numbers are listed here (see CSSProperty.js in React)
+
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface DOMAttributes<T extends EventTarget> extends svelte.JSX.DOMAttributes<T> {}
+
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface AriaAttributes extends svelte.JSX.AriaAttributes {}
+
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface HTMLAttributes<T extends EventTarget> extends svelte.JSX.HTMLAttributes<T> {
```

Comment:
The new transformation has no problem with colons in the attribute name and doesn't transform them, so they are listed here.